langroid/agent/special/neo4j/neo4j_chat_agent.py

```diff
@@ -398,7 +399,7 @@ def graph_schema_tool(
 
     def _init_tools_sys_message(self) -> None:
         """Initialize message tools used for chatting."""
-        self.tried_schema = False
+        self.tried_schema = self.tried_schema or False
```

Comment:
superfluous

Reply:
this prevents `self.tried_schema` from being false when it's already set to true after calling `_initialize_db` and there is a database exists